Manifests/ManifestsApple/com.apple.security.acme.plist

```diff
@@ -13,9 +13,12 @@
 	<key>pfm_ios_min</key>
 	<string>16.0</string>
 	<key>pfm_last_modified</key>
-	<date>2022-09-06T09:11:27Z</date>
+	<date>2023-02-27T14:18:42Z</date>
+	<key>pfm_macos_min</key>
+	<string>13.1</string>
 	<key>pfm_platforms</key>
 	<array>
+		<string>macOS</string>
 		<string>iOS</string>
```

Comment:
please sort these alphabetically.  I keep harping over a somewhat minor thing but it really helps with consistency of viewing